### PR TITLE
Le Miko has Arrived

### PR DESCRIPTION
**Pull request by Elijahrane**
## About the PR 
Adds the Shrine Maiden's Wig, Shrine Maiden's Outfit, and Gohei to the game. They are available in the theater vendor.
**Screenshots**
front view of miko
inhand-showcase
swept capitulate
drobe

**Changelog**

:cl: Rane
- add: The Red-and-White Shrine Maiden outfit complete with gohei is now available in the theater vendor!

### CODE REVIEW

**Review by Zumorica — APPROVED** [review]

File: Resources/Textures/Objects/Weapons/Melee/gohei.rsi/meta.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/799647e170c9b7dbdbb28bdf46234e012b6bd6c8",
+    "size": {
+        "x": 32,
+        "y": 32
+    },
+    "states": [
+        {
+            "name": "inhand-left",
+            "directions": 4,
+            "delays": [
```
Comment (SweptWasTaken) [inline Resources/Textures/Objects/Weapons/Melee/gohei.rsi/meta.json:12]: Remove delays here

File: Resources/Textures/Objects/Weapons/Melee/gohei.rsi/meta.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+                ],
+                [
+                    1.0
+                ]
+            ]
+        },
+        {
+            "name": "inhand-right",
+            "directions": 4,
+            "delays": [
```
Comment (SweptWasTaken) [inline Resources/Textures/Objects/Weapons/Melee/gohei.rsi/meta.json:30]: Remove

**Review by SweptWasTaken — CHANGES_REQUESTED** [review]

**Comment by SweptWasTaken** [comment]
You should be pushing from master. Make a new branch every time you make a PR to SS14.

**Comment by Elijahrane** [comment]
I will do that on future PRs

**Comment by PJB3005** [comment]
Merge this to spite swept.

**Comment by SweptWasTaken** [comment]
> Merge this to spite swept.

You're not spiting me you absolute tool there's literally a screenshot of me commenting on the PR that I approve.

**Pull request merged**